.travis.yml

```diff
@@ -39,17 +39,29 @@ jobs:
 
     - php: 7.2
       env: setup=lowest
+    - php: 7.2
+      env: illuminate=~6.0
+    - php: 7.2
+      env: illuminate=~7.0
```

Comment:
`illuminate/support v7.x` required php `v7.3` and above. 
See: https://github.com/illuminate/support/blob/master/composer.json#L17

Also `illuminate=~7.0` and `illuminate=~6.0` constraints with first minor version (`>=7.0.0 <7.1.0`)
See: https://jubianchi.github.io/semver-check/#/~7.0/7.1.0
Better to use `^7.0` or `7.*` version constraint

Reply:
1) https://github.com/illuminate/support/blob/7.x/composer.json#L17
2) Changed